src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -143,35 +143,40 @@ const SubmissionDetails = _.flow(
     ])
   }
 
+  /**
+   * If the text has multiple lines, return just the first line of text with ellipses to indicate truncation.
+   */
+  const firstLine = text => {
+    const lines = text ? text.split('\n') : []
+    return lines.length > 1 ? `${lines[0]} ...` : lines.length > 0 ? lines[0] : undefined
+  }
+
   /*
    * Page render
    */
   return div({ style: { padding: '1rem 2rem 2rem', flex: 1, display: 'flex', flexDirection: 'column' } }, [
     submissionDetailsBreadcrumbSubtitle(namespace, name, submissionId),
     _.isEmpty(submission) ? centeredSpinner() : h(Fragment, [
       div({ style: { display: 'grid', gridTemplateColumns: '1fr 4fr' } }, [
-        div({ style: { display: 'grid', gridTemplateRows: 'repeat(3, 33%)' } }, [
+        div({ style: { display: 'grid', gridAutoRows: 'min-content' } }, [
```

Comment:
I like your solution by restricting the comment to a single line in the display.
If UX and Product likes that as well, I would recommend:
```suggestion
        div({ style: { display: 'grid', gridTemplateRows: '1fr auto' } }, [
```
As this will give the status section and the comment section consistent heights. I think the UX of entering a multiline comment and then only seeing one line when there is a lot of whitespace underneath can make it feel like the comment wasn't saved correctly. If there is only enough space for 1 line, it may seem more natural to the user that their multiline comment was clipped.

Reply:
I like it! It solves both the problem of the comment field not aligning with anything, and any potential confusion about the truncation. Thanks for taking the time to play around with it!